.circleci/config.yml

```
@@ -1007,18 +1054,22 @@ workflows:
           context: consul-ci
           requires:
             - dev-upload-docker
+      - acceptance-kind-cni-1-23:
```

Also, we should use the same kind version as other acceptance tests in this pipeline
```suggestion
      - acceptance-tproxy-cni:
```